graph/patterns/change-tracking.md

```diff
@@ -0,0 +1,161 @@
+  `/me/planner/all/delta` – this returns changes to any resource within planner that a user is subscribed to as a heterogenous collection.
+
+- Although this capability is similar to the OData `$delta` feed capability, it is a different construct. Microsoft Graph APIs MUST provide change tracking through the delta function and MUST NOT implement the OData `$delta` feed when providing change tracking capabilities to ensure the uniformity of the API experience.
+- API producers might use `$skipToken` and `$deltaToken` within their implementations of `nextLink` and `deltaLink`, however the URLs are defined as being opaque and the existence of the tokens MUST NOT be documented.   It is not a breaking change to modify the structure of `nextLinks` or `deltaLinks`.- 
+- `nextLink` and `deltaLink` URLs are valid for a specific period before the client application needs to run a full synchronization again.For `nextLink`, a minimal validity time should be 1 hour. For `deltaLink`, a minimal validity time should be seven days. When a link is no longer valid it must return a standard error with a 410 GONE response code.
+  
+
+## Alternatives
+
+- Change notifications pattern with rich payloads – for use cases where API consumers would find calling back into Microsoft Graph onerous and absolute integrity guarantees are less critical.
+
+
+## Examples
+
+### Change tracking on entity set
+
+```
+<Function Name="delta" IsBound="true"> 
+     <Parameter Name="bindingParameter" Type="Collection(Microsoft.DirectoryServices.user)" /> 
+     <ReturnType Type="Collection(Microsoft.DirectoryServices.user)" /> 
+</Function> 
+<EntitySet Name="users" EntityType="Microsoft.DirectoryServices.user"> 
+    <Annotation Term="Org.OData.Capabilities.V1.ChangeTracking"> 
+      <Record> 
+        <PropertyValue Property="Supported" Bool="true" /> 
+      </Record> 
+    </Annotation> 
+</EntitySet> 
+```
+
+### Change tracking on navigation property
+
+```
+<Function Name="delta" IsBound="true">
+        <Parameter Name="bindingParameter" Type="Collection(microsoft.education.rostering.api.educationClass)" />
+        <ReturnType Type="Collection(microsoft.education.rostering.api.educationClass)" />
+      </Function>
+<EntityType Name="educationRoot">
+        <NavigationProperty Name="classes" Type="Collection(microsoft.education.rostering.api.educationClass)" ContainsTarget="true">
+          <Annotation Term="Org.OData.Capabilities.V1.ChangeTracking">
+            <Record>
+              <PropertyValue Property="Supported" Bool="true" />
+            </Record>
+          </Annotation>
+        </NavigationProperty>
+</EntityType>
+```
+
+### Delta payload
+
+ Here, a user resource is updated, and there is one user added to and one removed from that user’s directReports collection. Additionally, a second user is deleted. In this case, there are no further pages of change records currently available.
+
+```
+GET https://graph.microsoft.com/v1.0/users/delta
```

Comment:
Shouldn't this request have a $deltaToken (based on the $odata.deltaLink property returned previously) as you are showing deltas in the response?